CinemaTicketSalesSystem/CinemaTicketSalesSystem/App_Start/BundleConfig.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Web.Optimization;
+using System.Web;
+
+namespace CinemaTicketSalesSystem.App_Start
+{
+    public class BundleConfig
+    {
+        public static void RegisterBundles(BundleCollection bundles)
+        {
+            bundles.Add(new ScriptBundle("~/bundles/chosen").Include(
+                "~/Scripts/Chosen/chosen.jquery.min.js"));
```

Comment:
you can make one bundle from all scripts
or make different bundles if you use them in different places